ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -361,17 +374,18 @@ protected EngineStatus getInvalidBlockHashStatus() {
     return INVALID;
   }
 
-  protected Optional<JsonRpcResponse> validateForkSupported(
+  protected ValidationResult<NewPayloadValidationResult> validateForkSupported(
       final Object id, final EnginePayloadParameter payloadParameter) {
-    return Optional.empty();
+    return ValidationResult.valid();
   }
 
-  protected Optional<JsonRpcResponse> validateTransactions(
-      final EnginePayloadParameter blockParam,
-      final Object reqId,
+  protected ValidationResult<NewPayloadValidationResult> validateBlobs(
       final List<Transaction> transactions,
-      final Optional<List<String>> maybeVersionedHashParam) {
-    return Optional.empty();
+      final BlockHeader header,
+      final Optional<BlockHeader> maybeParentHeader,
+      final Optional<List<String>> maybeVersionedHashParam,
+      final ProtocolSpec protocolSpec) {
+    return ValidationResult.valid();
```

Comment:
nit: Since Cancun will be the new default, I think it is preferable to put the actual blob validation code in the Abstract class and override with valid() for V1 and V2.

That means when we no longer want to support V2, it's a simple delete of EngineNewPayloadV2 class without any refactoring.

For example, this was done for V1 with these two overrides https://github.com/hyperledger/besu/blob/main/ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV1.java#L44-L52
(indeed, we could probably delete those V1 classes now that Shanghai is live)